src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -96,7 +96,7 @@ public static class AdditionalMetadataReferences
                     "net6.0",
                     new PackageIdentity(
                         "Microsoft.NETCore.App.Ref",
-                        "6.0.0-preview.6.21352.12"),
+                        "6.0.0-rc.1.21451.13"),
```

Comment:
The new `Message` and `Url` properties on `RequiresPreviewFeatureAttribute` shipped in RC1